gallery/visualtests/nxDrawer.js

```diff
@@ -134,18 +134,28 @@ describe('NxDrawer', function() {
       await checkFullPageScreenshot();
     });
 
-    it('it stays open when the toast is closed', async function() {
+    it('stays open when the toast is closed', async function() {
       await getPage().setViewport(viewportSize);
-      const [openErrorToastButton] = await waitAndGetElements('#nx-toast-error-open-button');
-      const drawer = await waitAndGetElements('#nx-drawer-with-nx-toast');
+
+      const [drawer] = await waitAndGetElements('#nx-drawer-with-nx-toast');
 
       await openDrawer(buttonId, drawerId);
```

Comment:
Just my preference, but maybe put this above the waitAndGetElements for `nx-drawer-with-nx-toast`. IMO it makes more sense for readability to open the drawer, then get the element. But works either way

Reply:
> Just my preference, but maybe put this above the waitAndGetElements for `nx-drawer-with-nx-toast`. IMO it makes more sense for readability to open the drawer, then get the element. But works either way

That makes sense: 95752e9a5cf87abd4d87c552048814cf10dc28f3